pkg/sql/sem/tree/stmt.go

```diff
@@ -151,6 +151,15 @@ func (l StatementList) Format(buf *bytes.Buffer, f FmtFlags) {
 	}
 }
 
+// ObserverStatement is a pseudo-interface to be implemented by
```

Comment:
naming nit: pseudo-interface? I would call this a marker interface. Take or leave, especially if we call other things pseudo-interface in the code :)